storage/tianmu/core/engine.cpp

```diff
@@ -1094,6 +1090,7 @@ void Engine::RenameTable([[maybe_unused]] Transaction *trans_, const std::string
   filter_cache.RemoveIf([id](const FilterCoordinate &c) { return c[0] == int(id); });
   system::RenameFile(tianmu_data_dir / (from + common::TIANMU_EXT), tianmu_data_dir / (to + common::TIANMU_EXT));
   RenameRdbTable(from, to);
+  DeleteTableIndex(from, thd);
```

Comment:
we should add some fault-point tests. DBUG_EXECUTE_IF(xxxx). we should make the drop operation is robust operation.

Reply:
added it